[For 2.6.3] Prepare new stable release 2.6.3 (#2848)
tracim/tracim: backend/doc/setting.md

Don't forget to set `website.base_url` and `website.title` for the frontend, as some features use them to
link the frontend in emails.

> :warning: its necessary to check if your smtp configuration working good before using tracim.

Reviewer: ```suggestion
:warning: It is necessary to check if your SMTP configuration is working correctly before using Tracim.
```

Author: Done

> In next release we purpose a quick solution to test if your smtp configuration work properly.

Reviewer: ```suggestion
In the next release we will include a quick solution to test if your STMP configuration works properly.
```

Author: Done
